### ISSUE TITLE
Thumbnail preview doesn't work for videos played from a network drive

### ISSUE THREAD

**Issue by adamwinn** [iina/iina#3408]
**System and IINA version:**
- macOS 11.3.1
- IINA 1.1.2

**Expected behavior:**
Video thumbnails should show for all videos, including videos played from a network drive

**Actual behavior:**
Video thumbnails only appear for videos played from the local drive

**Steps to reproduce:**
Place a video on a network drive.  (I use a NAS connected via SMB)
Play the video
Notice how thumbnails do no appear

**How often does this happen?**
Every time

**Comment by adamwinn** [comment]
It looks like `defaults write com.colliderli.iina enableThumbnailForRemoteFiles -bool YES` fixes this.  I hope it can be added to the UI preferences

**Comment by jollino** [comment]
Agreed, I understand the reasoning behind this default (not all network shares are happy with requests like that) but it would be nice to have this as a setting in the Preferences window. :-)

**Comment by realav** [comment]
Thumbnails seek previews are **not working** for network and HTTP web streams even after using `defaults write com.colliderli.iina enableThumbnailForRemoteFiles -bool YES`.
